admin_solutions/master_node_config.adoc

```diff
@@ -417,24 +425,34 @@ and substitute in whatever values you want to change.
 |===
 |Option |Description
 
-|`*nodeName*`
-|The value of the `*nodeName*` (string) is used to identify this particular node in the cluster. If possible, this should be your fully qualified hostname. If you are describing a set of static nodes to the master, then this value must match one of the values in the list.
-
-|`*nodeIP*`
-|A node may have multiple IPs. This specifies the IP to use for pod traffic routing. If left unspecified, a network look-up is performed on the nodeName, and the first non-loopback address is used.
-
-|`*servingInfo*`
-|Describes how to start serving.
+|`*allowDisabledDocker*`
+|If this is set to `true`, then the Kubelet will ignore errors from Docker. This means that a node can start on a machine that does not have Docker started.
 
-|`*masterKubeConfig*`
-|The filename for the .kubeconfig file that describes how to connect this node to the master.
+|`*authConfig*`
+|Holds authn/authz configuration options.
 
 |`*dnsDomain*`
 |Holds the domain suffix.
 
 |`*dnsIP*`
 |(string) Contains the IP. Can be controlled with `*openshift_dns_ip*` in the Ansible inventory.
 
+|`*dockerConfig*`
+|Holds Docker-related configuration options.
+
+|`*imageConfig*`
+|Holds options that describe how to build image names for system components.
+
+|`*iptablesSyncPeriod*`
+|(string) How often iptables rules are refreshed. This can be controlled with
+`*openshift_node_iptables_sync_period*` from the Ansible inventory.
+
+|`*kubeletArguments,omitempty*`
+|Key-value pairs that are passed directly to the Kubelet that matches the Kubelet's command line arguments. These are not migrated or validated, so if you use them, then they may become invalid. Use caution, because these values override other settings in the node configuration that may cause invalid configurations.
```

Comment:
"passed directly to the Kubelet that match the Kublet's command line arguments" ??? I disagree. I'm leaving this as is, @bfallonf